cli/src/oauth/mod.rs

```diff
@@ -89,7 +89,13 @@ async fn generate_token() -> String {
     } else if let Some(error) = json["error"].as_str() {
       if error != "authorization_pending" {
         eprintln!("Authentication failed. Please try again. Err: {}", error);
-        error!("Authentication failed. Err: {}", error);
+        client_error(
+          "",
+          "AUTH_FAILED",
+          Some(&format!("Authentication failed. Err: {}", error)),
+          None,
+        )
+        .await;
```

Comment:
hmm I think the endpoint will fail if the token is empty? we should only allow this error type to be allowed in the endpoint